app/code/Magento/Bundle/Test/Unit/Pricing/Price/ConfiguredPriceTest.php

```diff
@@ -0,0 +1,185 @@
+     */
+    private $amountInterfaceMock;
+
+    /**
+     * @var DiscountCalculator|MockObject
+     */
+    private $discountCalculatorMock;
+
+    /**
+     * Initialize base dependencies
+     */
+    protected function setUp(): void
+    {
+        $basePrice = $this->getMockForAbstractClass(PriceInterface::class);
+        $basePrice->expects($this->any())->method('getValue')->willReturn($this->basePriceValue);
+
+        $this->priceInfoMock = $this->createMock(Base::class);
+        $this->priceInfoMock->expects($this->any())->method('getPrice')->willReturn($basePrice);
+        $this->productMock = $this->getMockBuilder(Product::class)
+            ->setMethods(['getPriceInfo', 'getOptionById', 'getResource', 'getId'])
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->productMock->expects($this->once())->method('getPriceInfo')->willReturn($this->priceInfoMock);
+        $this->productMock->expects($this->any())->method('getId')->willReturn(123);
+
+        $this->itemMock = $this->getMockBuilder(ItemInterface::class)
+            ->getMock();
+        $this->itemMock->expects($this->any())->method('getProduct')->willReturn($this->productMock);
+
+        $this->priceCurrencyMock = $this->getMockForAbstractClass(PriceCurrencyInterface::class);
+
+        $this->jsonSerializerMock = $this->getMockBuilder(Json::class)
+            ->getMock();
+        $this->configuredPriceSelectionMock = $this->getMockBuilder(ConfiguredPriceSelection::class)
+            ->setMethods(['getSelectionPriceList'])
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->configuredPriceSelectionMock->expects($this->any())->method('getSelectionPriceList')
+            ->willReturn($this->prepareAndReturnSelectionPriceDataStub());
+        $this->amountInterfaceMock = $this->getMockBuilder(AmountInterface::class)->getMock();
+        $this->amountInterfaceMock->expects($this->any())->method('getBaseAmount')
+            ->willReturn(100.0);
+        $this->calculator = $this->getMockBuilder(Calculator::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->calculator->expects($this->any())->method('calculateBundleAmount')
+            ->willReturn($this->amountInterfaceMock);
+        $this->discountCalculatorMock = $this->getMockBuilder(DiscountCalculator::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->discountCalculatorMock->expects($this->any())->method('calculateDiscount')
+            ->willReturn(-5.0);
+        $this->model = new ConfiguredPrice(
+            $this->productMock,
+            1,
+            $this->calculator,
+            $this->priceCurrencyMock,
+            null,
+            $this->jsonSerializerMock,
+            $this->configuredPriceSelectionMock,
+            $this->discountCalculatorMock,
+        );
+        $this->model->setItem($this->itemMock);
```

Comment:
Same this, should be in `testGetValueMethod`, as it represents a different case that you cover in `testGetValueMethodNoItem`